app/src/components/GrantCard.vue

```diff
@@ -1,40 +1,37 @@
 <!-- GrantCard -->
 
 <template>
-  <figure class="group">
+  <figure class="group cursor-pointer">
```

Comment:
If we're using the `cursor-pointer` class here, we can probably remove the usage two lines below